core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -544,6 +615,18 @@ private void verifyPerBrokerConfigValue(Admin client,
         }, 5000, config + " are not updated");
     }
 
+    private void verifyDeletedGroupConfig(Admin client,
```

Comment:
According to https://github.com/apache/kafka/pull/16887#discussion_r1719827276, we are always able to see the group configs. Hence, we should check that the value is revert to "default value" after we delete the configs.